circle.yml

```diff
@@ -1,4 +1,17 @@
 machine:
   node:
     version: 4.3.0
-
+test:
+  override:
+    - ./node_modules/.bin/eslint --quiet:
```

Comment:
There's no way to pass args to a makefile target (that i'm aware of?), so ... not sure how else to do it. maybe move to a `npm run lint` which runs eslint and move the makefile to use that?